tools/cldr-apps/js/src/views/VettingSummary.vue

```diff
@@ -70,7 +70,7 @@
             :key="locale"
           >
             <td>
-              <tt>{{ locale }}—{{ humanizeLocale(locale) }}</tt>
+              <code>{{ locale }}—{{ humanizeLocale(locale) }}</code>
```

Comment:
we started getting "[Vue warn]: Failed to resolve component: tt at <VettingSummary >" recently -- tt is obsolete in html5 -- see https://www.w3schools.com/tags/tag_tt.asp